src/shared/src/types/reimbursement-requests-types.ts

```diff
@@ -1,4 +1,87 @@
+import { WbsNumber } from './project-types';
+import { User } from './user-types';
+
 export enum Club_Account {
```

Comment:
can you rename this to ClubAccount while you’re here and fix the corresponding errors if there are any